papermage/magelib/document.py

```diff
@@ -55,6 +65,7 @@ def annotate_entity(self, field_name: str, entities: List[Entity]) -> None:
 
         setattr(self, field_name, entities)
         self.__entity_span_indexers[field_name] = EntitySpanIndexer(entities=entities)
+        self.__entity_box_indexers[field_name] = EntityBoxIndexer(entities=entities)
```

Comment:
What happens here if any/all of the entities have only boxes or spans?

Reply:
mmmm lemme test